frontend/app/templates/tasks.hbs

```diff
@@ -1,5 +1,9 @@
+{{navigation-bar session=session}}
```

Comment:
The issue is that we don't want a navigation bar neither in `index.hbs` nor in `login.hbs` (that I would put inside the `session.hbs`, I would like to work next on login / registration / forgot password / password recovery screens).

So we would include the navigation twice: `tasks.hbs` and `users.hbs`.

I have been investigating and I haven't found a way to use a different layout than `application.hbs`

But I am open to alternatives. How would you include the navigation in `application.hbs` but hide it in the frontpage (`index.hbs`) and login screens?